lib/Slic3r/GUI/Plater.pm

```diff
@@ -2432,6 +2432,11 @@ sub select_view {
     }
 }
 
+sub zoom{
+    my ($self, $direction) = @_;
+    $self->{canvas3D}->zoom($direction);
```

Comment:
I've updated my code. but there are two concerns:
* Applying any changes to 3d platter tab by Mouse or keyboard also affects the platter in the preview tab.
* Also There is no Mouse Zoom in the 2D platter tab.